### PR TITLE
services/horizon: Fix changelog for horizon 2.1.0

### PR DESCRIPTION
**Pull request by tamirms**

### CODE REVIEW

File: services/horizon/CHANGELOG.md [hunk 1]
```diff
@@ -19,14 +19,20 @@ file. This project adheres to [Semantic Versioning](http://semver.org/).
 
 **The migration will be performed by the ingestion system (through a state rebuild) and, thus, if some of your Horizon nodes are not ingestors (i.e. no `--ingestion` flag enabled) you may experience 500s in the `GET /claimable_balances/` requests until an ingestion node is upgraded. Also, it's worth noting that the rebuild process will take several minutes and no new ledgers will be ingested until the rebuild is finished.**
 
+### Breaking changes
+
+* Add a flag `--captive-core-storage-path/CAPTIVE_CORE_STORAGE_PATH` that allows users to control the storage location for Captive Core bucket data ([3479](https://github.com/stellar/go/pull/3479)).
```
Comment (bartekn) [inline services/horizon/CHANGELOG.md:24]: Maybe we should add a note that this was stored in `/tmp` before?

File: services/horizon/CHANGELOG.md [hunk 2]
```diff
@@ -19,14 +19,20 @@ file. This project adheres to [Semantic Versioning](http://semver.org/).
...
+  If the captive core storage path is not configured Horizon will default to using the current working directory.
+  Previously, the `TMPDIR` environment variable was used to configure the storage location. Now, `TMPDIR` is ignored.
+* Add a flag `--captive-core-log-path`/`CAPTIVE_CORE_LOG_PATH` that allows users to control the location of the logs emitted by Captive Core ([3472](https://github.com/stellar/go/pull/3472)). If you have a `LOG_FILE_PATH` entry in your Captive Core toml file remove that entry and use the horizon flag instead.
```
Comment (bartekn) [inline services/horizon/CHANGELOG.md:27]: `LOG_FILE_PATH` wouldn't it prevent Stellar-Core from starting (duplicate config entry)?

**Review by bartekn — APPROVED** [review]

File: services/horizon/CHANGELOG.md (hunk 2 above)
Comment (tamirms) [reply services/horizon/CHANGELOG.md:27]: yeah, that's why I asked them to remove `LOG_FILE_PATH` from the captive core toml file

Comment (jacekn) [reply services/horizon/CHANGELOG.md:27]: Yes exactly, anyone who has `LOG_FILE_PATH` set in the captive core toml will need to remove it

Comment (bartekn) [reply services/horizon/CHANGELOG.md:27]: Oh, I read it as: "Captive Core will remove that entry and use the horizon flag instead`. Nvm!

**Review by jacekn — APPROVED** [review]
LGTM, thank you for updating it!

**Review by bartekn — APPROVED** [review]

**Pull request merged**